raiden/raiden_service.py

```diff
@@ -656,7 +656,7 @@ def _synchronize_with_blockchain(self) -> None:
             contract_manager=self.contract_manager,
             last_fetched_block=last_block_number,
             event_filters=filters,
-            max_number_of_blocks_to_poll=BlockNumber(100_000),
+            max_number_of_blocks_to_poll=BlockNumber(1_000),
```

Comment:
I think it is reasonable to keep the code simple, unless we find the need to optimize during testing.
`_synchronize_with_blockchain` is only called during startup, so the performance hit of added rpc requests is acceptable IMO.

Reply:
for 3rd party ethereum rpc (i.e. infura) it may be a little more interesting, since this will add about `5` five extra rpc calls every day since deployment of the contracts:
```
blocks_per_day = (3600 * 24) / 15 = 5760
```